### ISSUE TITLE
Browser Hijack on Windows 11

### ISSUE THREAD

**Issue by facknrite** [ChrisTitusTech/win10script#289]
Chris you may want to  double check your script. I had to reset my computer 3 times after running your script on Windows 11. It creates some sort of backdoor and a random proxy script appeared on my windows settings that i couldn't get rid of it. It was pointing to http://127.0.0.1:86 
The only solution was to reinstall windows 11. I tried to re run your Windows 10 script and it happened again. Now i reset my computer one more time and didn't run your script and i haven't got any issues with it.
I don't know if it happens when removing the bloatware or when i run your Essential Tweaks since it just doesn't happen right away. Sometimes it takes a couple hours to appear.

**Comment by usrDottik** [comment]
isn't http://127.0.0.1 just localhost? How is that a browser Hijack?

**Comment by facknrite** [comment]
localhost is indeed 127.0.0.1 without a port number
The browser hijack adds a port number which you can't get rid of. https://127.0.0.1:86

**Comment by usrDottik** [comment]
did you try to repair the browser? I don't think any kind of malware disappears YTube's left sidebar, and if you can not enter to https://google.com check in C:\Windows\System32\Drivers\etc\hosts if you have google URL in it, no favourites, no searching history, are you sure you didn't wipe the browser data with this part or something similar? I don't know what kind of malware listens to localhost in port 86, yeah port 86 may have been used in the past for not-so-good purposes, but I guess it's a communication to the exterior, not to localhost, else, correct me

**Comment by usrDottik** [comment]
no, I think you didn't understand the part of communication to the exterior, with that I mean that it connected to an ip that is NOT localhost, regarding the google certificate being invalid, if you made the PC use UTC, check for TimeZone and synchronize, else certificate can be invalid thanks to it, no AV will detect a program listening to localhost as malware, since it's not something that you can take as a backdoor, it'd be a backdoor if the connection was to the exterior, not to the interior, like localhost

**Comment by facknrite** [comment]
I tried everything and this browser hijacked have been reported in other websites but without a final resolution.

**Comment by Carterpersall** [comment]
This is weird because there shouldn't be anything in the script that would cause this issue. It does seem to be the common denominator in your situation though so all I can do is recommend you try running this in a VM if you can to see if you get the issue, and if you don't, then it might not be the script that is causing this. If you do, then more research will need to be done into what exactly is causing this.

**Comment by usrDottik** [comment]
also @facknrite, remember me? Well if you do, I ran the script, but I couldn't reproduce the issue :p, I don't see any program that starts up with PC to make such effects like a backdoor, nor an app allowed though firewall, and I didn't get any proxy in settings that uses http://127.0.0.1:86 as proxy, if u could send sshot of what shows so I can try and sort of replicate it

**Comment by fgclue** [comment]
It didn't remove the YT Sidebar, It was just a bug, also did you remove cortana? If so that's probally why you can't search.

**Comment by Carterpersall** [comment]
@facknrite any updates on the issue?
